include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2250,6 +2250,15 @@ static inline int bt_le_whitelist_clear(void)
 /**
  * @brief Set (LE) channel map.
  *
+ * Channel map format is chan_map[5] = {0xXX, 0xXX, 0xXX, 0xXX, 0xXX};
+ * chan_map[0] = 39-32
+ * chan_map[1] = 31-24
+ * chan_map[2] = 23-16
+ * chan_map[3] = 15-8
+ * chan_map[4] = 7-0
```

Comment:
We should specify the mapping more precisely. I.e. What `chan_map[4] = 7-0` means. I guess it's `chan_enabled[ch] = 0!= (chan_map[4 - (ch / 8)] & (1 << (ch % 8)))`?

Reply:
It would have been easier if the function had taken an `uint64_t` value instead of a `uint8_t [5]`. 

w.r.t. @alwa-nordic comment, I actually think the suggest form is easier to read that what @alwa-nordic suggests, although I would consider reversing the order: 
```c
 * chan_map[0] = 32-39
 * chan_map[1] = 24-31
 * chan_map[2] = 16-23
 * chan_map[3] = 8-15
 * chan_map[4] = 0-7
```

Reply:
I didn't mean to suggest that we put what I wrote in the documentation. I agree, it's horrible to read :P. I just wanted to confirm here what the relation is.

I think we should add "The lowest bit of `chan_map[4]` is channel 0.", and the rest becomes obvious. Or is it clear that "0-7" is from lowest to highest bit (and not the other way around)?